src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java

```diff
@@ -30,7 +28,6 @@
 
 @Service
 public class FormaliseResponseAcceptanceService {
-    private static final Logger logger = LoggerFactory.getLogger(FormaliseResponseAcceptanceService.class);
 
```

Comment:
Note to reviewers: this was cherry-picked from https://github.com/hmcts/cmc-claim-store/pull/533 to fix the sonar complaint, though the issue wasn't introduced by either - can be ignored.